resources/assets/js/components/Twitter.vue

```diff
@@ -74,14 +74,14 @@
         data() {
             return {
                 displayingTopTweetSince: new moment(),
```

Comment:
Can all occurences of `new moment()` be replaced by `moment()`? Feels cleaner to use it as a function.